src/core_plugins/kibana/server/tutorials/apm/apm_client_instructions.js

```diff
@@ -0,0 +1,67 @@
+export const NODE_CLIENT_INSTRUCTIONS = [
+  {
+    title: 'Install the APM agent',
+    textPre: 'Install the APM agent for Node.js as a dependency to your application.',
+    commands: [
+      'npm install elastic-apm-node --save'
+    ]
+  },
+  {
+    title: 'Configure the agent',
+    textPre: 'Agents are libraries that run inside of your application process.' +
+      ' APM services are created programmatically based on the `serviceName`.' +
+      ' This agent supports Express, Koa, hapi, and custom Node.js.',
+    commands: [
+      `// Add this to the VERY top of the first file loaded in your app`,
+      `var apm = require('elastic-apm-node').start({curlyOpen}`,
+      `  // Set required service name (allowed characters: a-z, A-Z, 0-9, -, _, and space)`,
+      `  serviceName: '',`,
+      `  `,
+      `  // Use if APM Server requires a token`,
+      `  secretToken: '',`,
+      `  `,
+      `  // Set custom APM Server URL (default: http://localhost:8200)`,
+      `  serverUrl: ''`,
+      `{curlyClose})`,
+    ],
+    textPost: 'See [the documentation]({config.docs.base_url}guide/en/apm/agent/nodejs/current/index.html)' +
+    ' for advanced usage. Babel users, please refer to [the documentation]' +
+    '({config.docs.base_url}guide/en/apm/agent/nodejs/current/advanced-setup.html#es-modules).'
+  }
+];
+
+export const DJANGO_CLIENT_INSTRUCTIONS = [
```

Comment:
@nreese I see that Flask, Rails, Rack and JS agent instructions are missing. I can try to add those if you want. Would be great if I could run this fork locally.

Reply:
I will add these this afternoon. Just wanted to get enough the the manifest file out so we could have a conversation and find any sticky points.